packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -69,7 +76,7 @@ const WizardHeading = ({ real_account_signup_target, currency, is_isle_of_man_re
                 country_standpoint.is_other_eu ||
                 country_standpoint.is_rest_of_eu
             ) {
-                return <Localize i18n_default_text='Add a real Deriv Multipliers account' />;
+                return <Localize i18n_default_text={`Add a real ${signup_modal_real_account_title} account`} />;
```

Comment:
Translation text extraction won't be proper with string interpolation.

Values should be passed In values param
```
<Localize
        i18n_default_text={'Add a real {{title}} account'}
        values={{
            title: signup_modal_real_account_title
        }}
    />
   ```

Reply:
fixed